JitsiConference.js

```diff
@@ -80,6 +84,7 @@ function JitsiConference(options) {
     this.connectionIsInterrupted = false;
 
 }
+JitsiConference.prototype.constructor = JitsiConference;
```

Comment:
I see. Can you add a comment so someone can tackle that sooner rather than later? :-)